doc/rados/operations/monitoring.rst

```diff
@@ -345,33 +345,70 @@ the following::
+The output of ``ceph df`` looks like this::
 
-- **CLASS:** The class of OSD device (or the total for the cluster)
+   CLASS  SIZE     AVAIL    USED     RAW USED  %RAW USED
+   ssd    202 GiB  200 GiB  2.0 GiB   2.0 GiB       1.00
+   TOTAL  202 GiB  200 GiB  2.0 GiB   2.0 GiB       1.00
+
+   --- POOLS ---
+   POOL                   ID  PGS  STORED   (DATA)   (OMAP)   OBJECTS  USED     (DATA)  (OMAP)   %USED  MAX AVAIL  QUOTA OBJECTS  QUOTA BYTES  DIRTY  USED COMPR  UNDER COMPR
+   device_health_metrics   1    1  242 KiB   15 KiB  227 KiB        4  251 KiB  24 KiB  227 KiB      0    297 GiB  N/A            N/A              4         0 B          0 B
+   cephfs.a.meta           2   32  6.8 KiB  6.8 KiB      0 B       22   96 KiB  96 KiB      0 B      0    297 GiB  N/A            N/A             22         0 B          0 B
+   cephfs.a.data           3   32      0 B      0 B      0 B        0      0 B     0 B      0 B      0     99 GiB  N/A            N/A              0         0 B          0 B
+   test                    4   32   22 MiB   22 MiB   50 KiB      248   19 MiB  19 MiB   50 KiB      0    297 GiB  N/A            N/A            248         0 B          0 B
+
+
+
+
+
+- **CLASS:** for example, "ssd" or "hdd"
 - **SIZE:** The amount of storage capacity managed by the cluster.
 - **AVAIL:** The amount of free space available in the cluster.
-- **USED:** The amount of raw storage consumed by user data.
+- **USED:** The amount of raw storage consumed by user data (excluding BlueStore's database)
 - **RAW USED:** The amount of raw storage consumed by user data, internal overhead, or reserved capacity.
 - **%RAW USED:** The percentage of raw storage used. Use this number in
   conjunction with the ``full ratio`` and ``near full ratio`` to ensure that 
   you are not reaching your cluster's capacity. See `Storage Capacity`_ for 
   additional details.
 
+
+**POOLS:**  
+
 The **POOLS** section of the output provides a list of pools and the notional 
 usage of each pool. The output from this section **DOES NOT** reflect replicas,
 clones or snapshots. For example, if you store an object with 1MB of data, the 
 notional usage will be 1MB, but the actual usage may be 2MB or more depending 
-on the number of replicas, clones and snapshots.
+on the number of replicas, clones and snapshots.  
 
-- **NAME:** The name of the pool.
-- **ID:** The pool ID.
-- **USED:** The notional amount of data stored in kilobytes, unless the number 
-  appends **M** for megabytes or **G** for gigabytes.
-- **%USED:** The notional percentage of storage used per pool.
-- **MAX AVAIL:** An estimate of the notional amount of data that can be written
-  to this pool.
+- **ID:** The number of the node within the pool.
+- **STORED:** actual amount of data user/Ceph has stored in a pool. This is similar to the USED column in earlier versions of Ceph but the calculations (for BlueStore!) are more precise (gaps are properly handled).
+- **(DATA):**
```

Comment:
data and omap are different kinds of information that can be stored in a rados object. data is your typical usage, for rbd, cephfs file data and rgw object data.

omap is key/value pairs - in practice this is primarily used for metadata storage by cephfs and rgw

Reply:
the reason they're separated here is to be able to see how much of each type you have. for bluestore this is particularly important because omap is stored in bluestore's db device, which is usually smaller if it is a separate disk (e.g. a common configuration is db on nvme, data on hdd).

Reply:
I've added this. There are still a couple of empty fields, but we can fill them during our weekly meeting on your Thursday and my Friday.